packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -358,7 +358,7 @@ export class RestAPIClient {
 			);
 		}
 
-		if ( response.errors === [] ) {
+		if ( response.errors.length > 0 && ! response.sent ) {
```

Comment:
**question:blocking** - reading the description of this PR it is unexpected to see a change here. Was it failing tests?

Reply:
Agreed with @worldomonation. It seems like a change that can (and should) be moved to another PR.